app/validators/return-logs/setup/period-used.validator.js

```diff
@@ -55,10 +55,14 @@ function _fromDateBeforeToDate(value, helpers) {
 }
 
 function _fullDate(day, month, year) {
+  if (!year && !month && !day) {
```

Comment:
Just wanted to check that the logic here is correct -- it only returns `null` if _all_ of these are falsy?